start_ocean_local_parity_node.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
awesome, good call with some bash scripts. But these look like a lot of duplication and we could just use one script, `start_ocean.sh`, and do the rest with options, like:

```
start.sh
start.sh --no-pleuston
start.sh --local-parity
```

Reply:
Good point, so it is.